src/components/ProtectedImage.vue

```diff
@@ -1,15 +1,15 @@
 <script lang="ts" setup>
 import filters from "@/utils/filters";
-import { useAuth0 } from "@auth0/auth0-vue";
-import { onMounted, ref } from "vue";
+import { AUTH0_INJECTION_KEY } from "@auth0/auth0-vue";
+import { inject, onMounted, ref } from "vue";
 
 const props = defineProps<{
   src: string;
 }>();
 
 const source = ref("");
 
-const { getAccessTokenSilently } = useAuth0();
+const { getAccessTokenSilently } = inject(AUTH0_INJECTION_KEY)!;
```

Comment:
to be honest I think the previous way was easier to understand. What's the benefit of the changes in this PR?